packages/account/src/Sections/Security/Passwords/passwords.jsx

```diff
@@ -67,4 +68,4 @@ const Passwords = observer(() => {
     );
 });
 
-export default Passwords;
+export default withRouter(Passwords);
```

Comment:
Currently in prod, The strings are not translated when language is changed.

Reply:
This is to fix a prod bug where page is not translated immediately after changing the language, and a manual refresh is required to reflect the updated language.

Reply:
Ideally, changing a language should trigger a rerender from the root node and update all the texts to chosen language. Adding `withRouter` might fix the issue on this page but this issue could happen on other pages too.

Reply:
I have removed the withRouter from the component and added current-language to the key of Routes so that component will rerender when language is changed.